.flake8

```diff
@@ -20,6 +20,28 @@ per-file-ignores =
     # F841 local variable assigned to but never used
     # E741 ambiguous variable name 'l'
     __init__.py: F401, F403
+    # HTML
+    man/build_html.py: E501
+    # wxgui
```

Comment:
There is a "gui" block below. Move this there. Also, some ignores there are no longer needed.

Reply:
I wanted reviewers' input on the files I included in the .flake8 configuration file. I added this way to keep track of whether I make chages on files or listing in the .flake8 configuration file. Could these files be listed in per-file-ignores or 88 line length enforced? I will delete duplicates if they are okay to be listed in per-file-ignore.